src/HotChocolate/Core/test/Types.Tests/Types/Descriptors/Conventions/XmlDocumentationProviderTests.cs

```diff
@@ -95,6 +95,22 @@ public void When_type_has_description_then_it_it_resolved()
             Assert.Equal("I am the most base class.", description);
         }
 
+        [Fact]
+        public void When_we_use_custom_documentation_files_they_are_correctly_loaded()
```

Comment:
This test will use the xml file with the custom name instead of the standard docs file.